todo_app/data/trello_items.py

```diff
@@ -14,38 +11,15 @@ def get_items_all():
         
     response_json = response.json()
 
-    return response_json
-
-def get_items(status):
-
     list_of_cards =[]
 
-    for trello_list_name in get_items_all():
+    for trello_list_name in response_json:
         cards = trello_list_name["cards"]
 
         for card in cards:
-            if trello_list_name["name"] == status:
-                list_of_cards.append(Item.from_trello_card(card, trello_list_name))
-                                                                  
-    return list_of_cards
-
-def get_open_items():
-    
-    list_of_cards_open = get_items('To Do')
-                                                                  
-    return list_of_cards_open
-
-def get_in_progress_items():
-    
-    list_of_cards_in_progress = get_items('Doing')
-                                                                  
-    return list_of_cards_in_progress
+            list_of_cards.append(Item.from_trello_card(card, trello_list_name))
 
-def get_closed_items():
-    
-    list_of_cards_closed = get_items('Done')
-                                                                  
-    return list_of_cards_closed
+    return list_of_cards
```

Comment:
Thanks Bogdan. Yes, I had only half refactored my code before submitting module 5 - I didn't realise it would be re-assessed as part of module 5. Good to know for next time. I've followed your advice and simplified the code as to your suggestion and it is all working fine. Thanks. Now to resolve the failing tests...